src/watchmaker/workers/salt.py

```diff
@@ -774,10 +774,22 @@ def _configuration_validation(self):
                     'parameter `git_repo` was not provided.'
                 )
 
+    def _check_salt_version(self):
```

Comment:
This method is currently in `SaltLinux`, which means it is only available to Linux systems. We need this check on Windows also. So let's move this method to `SaltBase`.